assignment2/q1.py

```diff
@@ -0,0 +1,36 @@
+# Print AncestorsGiven a Binary Tree and a key, 
+# write a function that prints all the ancestors of the key in the givenbinary tree.
+class Node:
+ 
+    
+    def __init__(self, data):
+        self.data = data 
+        self.left = None
+        self.right = None
+
+    def printAncestors(root, target):
+
```

Comment:
Hi @windo @luxiya01 
I tried a recursive solution.The errors appeared when I copy paste the solution from vs code to github directly.
I will take a look again at the solution.